src/data_structures.rs

```diff
@@ -127,32 +127,28 @@ impl<F: PrimeField, PC: PolynomialCommitment<F, DensePolynomial<F>>> Proof<F, PC
 
     /// Prints information about the size of the proof.
     pub fn print_size_info(&self) {
-        use ark_poly_commit::{PCCommitment, PCProof};
+        use ark_poly_commit::PCCommitment;
 
-        let size_of_fe_in_bytes = F::zero().into_repr().as_ref().len() * 8;
         let mut num_comms_without_degree_bounds = 0;
         let mut num_comms_with_degree_bounds = 0;
         let mut size_bytes_comms_without_degree_bounds = 0;
         let mut size_bytes_comms_with_degree_bounds = 0;
-        let mut size_bytes_proofs = 0;
         for c in self.commitments.iter().flat_map(|c| c) {
             if !c.has_degree_bound() {
                 num_comms_without_degree_bounds += 1;
-                size_bytes_comms_without_degree_bounds += c.size_in_bytes();
+                size_bytes_comms_without_degree_bounds += c.serialized_size();
             } else {
                 num_comms_with_degree_bounds += 1;
-                size_bytes_comms_with_degree_bounds += c.size_in_bytes();
+                size_bytes_comms_with_degree_bounds += c.serialized_size();
             }
         }
```

Comment:
Maybe we just write this as `self.commitments.serialized_size()`? And then separately report the number of degree-bounded and non-degree-bounded commitments.